app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+    <div class="container">
+        <%= link_to "sample app", "#", id: "logo" %>
+            <nav>
+                <ul class="nav navbar-nav navbar-right">
+                    <li><%= link_to t("Home"), "#" %></li>
```

Comment:
`"Home"` tên biến ko bắt đầu bằng ừ in hoa nhé, self check các file html bên dưới